release_notes/ocp-4-3-release-notes.adoc

```diff
@@ -121,6 +121,16 @@ the {product-title} web console.
 The `KubeletTooManyPods` alert now uses the Pod capacity metric as a threshold
 instead of a fixed number.
 
+[id="ocp-4-3-cluster-logging"]
+=== Cluster logging
```

Comment:
Is this the right spot for this? There are two monitoring-related items below this that now look like they fall under cluster logging.

Reply:
Good catch! Thank you @bergerhoffer